v-next/hardhat-utils/package.json

```diff
@@ -26,7 +26,8 @@
     "./number": "./dist/src/number.js",
     "./package": "./dist/src/package.js",
     "./request": "./dist/src/request.js",
-    "./string": "./dist/src/string.js"
+    "./string": "./dist/src/string.js",
+    "./sub-process": "./dist/src/sub-process.js"
```

Comment:
This is very nitpicky, but i think most people consider `subprocess` a single word

Reply:
[Done here](https://github.com/NomicFoundation/hardhat/pull/5471/commits/a1354fca15ac2aa95bc5082f157cc5338bcc29c5)

Reply:
you should rename the function name as well

Reply:
it's still `sub-process` in the package.json

Reply:
resolving and moving the conversation to the new version of this file